lib/benchee/benchmark/runner.ex

```diff
@@ -225,11 +225,15 @@ defmodule Benchee.Benchmark.Runner do
        ) do
     {run_time, memory_usage} = measure_iteration(scenario, scenario_context)
 
-    if is_nil(memory_usage) do
-      {run_time / num_iterations, memory_usage}
-    else
-      {run_time / num_iterations, memory_usage / num_iterations}
-    end
+    adjust_for_iterations(run_time, memory_usage, num_iterations)
+  end
+
+  defp adjust_for_iterations(run_time, nil, num_iterations) do
+    {run_time / num_iterations, nil}
+  end
+
+  defp adjust_for_iterations(run_time, memory_usage, num_iterations) do
+    {run_time / num_iterations, memory_usage / num_iterations}
```

Comment:
@devonestes it tests this right here :) Or well the line above. That we actually divide by `num_iterations` - I errornously introduced a version where I didn't divide by num_iterations and no test failed. The test you mentioned will faill if we remove it because (at least on my machine) it gets a factor 100 so memory consumption will be much higher if we don't divide.